.github/actions/templates/avm-validateModuleDeployment/action.yml

```diff
@@ -77,6 +77,7 @@ runs:
             'avm/res/azure-stack-hci/cluster'                    # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/compute/image'                              # Failing on resource deletion when trying to delete RBAC at subscription level
             'avm/res/compute/disk'                               # Failing on resource deletion when trying to delete RBAC at subscription level
+            'avm/res/kubernetes/connected-clusters'              # Failing on resource deletion when trying to delete RBAC at subscription level
```

Comment:
Is this exception intended? Could you please share a pipeline run for this failing using OIDC?